icu4j/main/collate/src/test/java/com/ibm/icu/dev/test/collator/AlphabeticIndexTest.java

```diff
@@ -374,7 +373,7 @@ public void TestInflow() {
             String printList = Arrays.asList(test).toString();
             assertEquals(LabelType.UNDERFLOW + "\t" + printList, 1, counter.get(LabelType.UNDERFLOW));
             assertEquals(LabelType.INFLOW + "\t" + printList, expected, counter.get(LabelType.INFLOW));
-            if (expected != counter.get(LabelType.INFLOW)) {
+            if (expected == counter.get(LabelType.INFLOW)) {
```

Comment:
I think this wants to remain as is: When the preceding assertion fails, then this code prints stuff "for debugging". It might effectively not work anymore since JUnit assertEquals() throws an exception, but we should at least keep the intent intact.

Reply:
Sorry, my bad.
I used it to force a printout.
Did it with the old code, new code, and compared that it is the same.
Fixed.